draft-ietf-webtrans-http3.md

```diff
@@ -113,23 +113,17 @@ closed.
 ## Establishing a Transport-Capable HTTP/3 Connection {#establishing}
 
 In order to indicate support for WebTransport, both the client and the server
-MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
-frame.  The SETTINGS_ENABLE_WEBTRANSPORT parameter value SHALL be either "0" or
-"1", with "0" being the default; an endpoint that receives a value other than
-"0" or "1" MUST close the connection with the H3_SETTINGS_ERROR error code.
+MUST send a SETTINGS_WEBTRANSPORT_MAX_SESSIONS value greater than "0" in their
+SETTINGS frame.  The default value for the SETTINGS_WEBTRANSPORT_MAX_SESSIONS
+parameter is "0", meaning that the endpoint is not willing to receive any
+WebTransport sessions.
 
 The client MUST NOT send a WebTransport request until it has received the
 setting indicating WebTransport support from the server.  Similarly, the server
 MUST NOT process any incoming WebTransport requests until the client settings
-have been received, as the client may be using a version of WebTransport
+have been received, as the client may be using a version of the WebTransport
```

Comment:
Should this paragraph be in a "to be removed by RFC editor" portion?

Reply:
Makes sense, done in https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/129/commits/6b437df3129ac03d6ef429f5e1dfc8785f78ffdd